### PR TITLE
Added a Few Guides

### PR DESCRIPTION
**Pull request by SonofForehead**
- Added Vehicle Guide
- Added Vehicles Wheels Guide
- Added Seat Guide
- Updated Clipboard Guide
- Updated Hinge Joint Door Guide

Sorry I made a new one I was changing the branch because I maaaay have done it on main

### CODE REVIEW

File: content/index.md
Comment (SonofForehead) [inline content/index.md:1]: Oops

File: package-lock.json
Comment (SonofForehead) [inline package-lock.json:1]: Oops

Comment (SonofForehead) [inline package-lock.json:1]: ???????????

Comment (6543ozne) [reply package-lock.json:1]: i hope yours gets merged!

**Review by MrBugo — APPROVED** [review]
LGTM

**Comment by 6543ozne** [comment]
Yay!

**Pull request merged**